File: Chat-Web/src/main/resources/static/Chat/src/types/WorkSpace.ts
export interface WorkSpace {
  todayNewUser: number;
  totalUser: number;

  totalMsg: number;
  todayNewMsg: number;
  onlineUser?: number;
  date?: string;
}
export interface userActiveVO {
  userID: string;
  nickName: string;
  icon: string;
  active: number;
}

export interface UserTotalActiveVO{
  userID: string;
  nickName: string;
  icon: string;
  totalActive: number;
  rank: number;
}

File: Chat-Common/src/main/java/com/summit/chat/model/entity/mysql/WorkSpace.java
package com.summit.chat.model.entity.mysql;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class WorkSpace implements Serializable {

  private long id;
  private long todayNewUser;
  private long totalUser;
  private Long totalMsg;
  private Long todayNewMsg;
  private Integer onlineUser;
  private String date;
  private java.sql.Timestamp createTime;


}
